openapi.yaml

```diff
@@ -160,57 +175,26 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/TransactionHistoryItem'
-    Question:
-      type: object
-      properties:
-        title:
-          type: string
-        answers:
-          type: array
-          items:
-            type: string
-    MultiChoicePoll:
+    HTTPPoll:
```

Comment:
Can we simply call it “Challenge”? The fact that it’s http isn’t relevant and “Poll” is too specific.

Reply:
@Elaz  @doodyparizada  maybe `HTMLPoll` ? I think the "Poll" postfix it's ok.